tsdoc/src/parser/NodeParser.ts

```diff
@@ -390,6 +398,7 @@ export class NodeParser {
       tokenReader.peekTokenAfterKind() === TokenKind.AtSign) {
       return undefined;
     }
+    tokenReader.readToken(); // read the "{"
```

Comment:
Capture the start marker here before reading the `{`, per my comment above.
```suggestion
    const startMarker: number = tokenReader.createMarker();
    tokenReader.readToken(); // read the "{"
```